srcpkgs/starship/template

```diff
@@ -1,18 +1,19 @@
 # Template file for 'starship'
 pkgname=starship
-version=0.51.0
+version=0.55.0
 revision=1
 build_style=cargo
 build_helper=qemu
-hostmakedepends="pkg-config libgit2-devel"
-makedepends="openssl-devel libgit2-devel"
+hostmakedepends="libgit2-devel pkg-config"
+makedepends="libgit2-devel openssl-devel"
+depends="git"
 checkdepends="git fish-shell"
 short_desc="Minimal, fast and customizable cross-shell prompt"
-maintainer="Aluísio Augusto Silva Gonçalves <aluisio@aasg.name>"
+maintainer="cinerea0 <cinerea0@protonmail.com>"
```

Comment:
Have you asked @AluisioASG about it? Seems unactive since Jun 2020, but who knows :p

Reply:
I believe they now use NixOS, but you're right that it couldn't hurt to ask. @AluisioASG: Would it be alright if I took maintainership of this package?